src/Cli/ConfigGenerator.cs

```diff
@@ -96,7 +97,16 @@ public static bool TryCreateRuntimeConfig(InitOptions options, out string runtim
                             "it does not support REST yet.");
                     }
 
+                    // If the option --rest.base-route is specified for cosmosdb_nosql, log a warning because
+                    // rest is not supported for cosmosdb_nosql yet.
+                    if (!string.Empty.Equals(restBaseRoute))
+                    {
+                        _logger.LogWarning("Configuration option --rest.base-route is not honored for cosmosdb_nosql since " +
+                            "it does not support REST yet.");
```

Comment:
```suggestion
                        _logger.LogWarning("Configuration option --rest.base-route is not honored for cosmosdb_nosql since it does not support REST.");
```

No need to split this over multiple lines.

Also dropped `yet` as that implies it's something we will end up supporting but there's no guarantee that we are going to support it.

Reply:
removed code.